### ISSUE TITLE
Update vertical item position on 'dragmove' to match user's mouse position

### ISSUE THREAD

**Issue by diegocasmo** [namespace-ee/react-calendar-timeline#738]
**Is your feature request related to a problem? Please describe.**
While a user is dragging an item to another group, the item's vertical position doesn't match the user's mouse position, which results in a non-ideal UX. 

This behavior can be seen [in this .gif](https://user-images.githubusercontent.com/4553097/79511963-09280c00-8006-11ea-8ad0-6a8ee5c95a9b.gif), where the item vertical position "jumps" from group to group due to its vertical position only being updated once inside the other group (and not while on 'dragmove'). This is particularly bad when multiple items belong to the same group and have a similar start/end time. 

[Here's another .gif](https://user-images.githubusercontent.com/4553097/79512788-bf402580-8007-11ea-8e4c-5ff80768fcbf.gif) showing how another React timeline library looks like when performing the same action. Notice how the vertical position of the item matches the user's mouse position. 

**Describe the solution you'd like**
Update the item position while it's being moved (just like it's now done horizontally), but also for vertical movements.

**Describe alternatives you've considered**
Would it be possible to include some type of position information on interact's [`dragmove` event](https://github.com/namespace-ee/react-calendar-timeline/blob/master/src/lib/items/Item.js#L260)?

**Additional context**
N/A

**Comment by Ilaiwi** [comment]
@diegocasmo This is how the drag and drop system works in this library. To fix this, this would need a big refactor for the drag and drop system that we are not planing to do anytime soon

**Comment by diegocasmo** [comment]
@Ilaiwi 

Got it! I took a look at the source code and you are correct. I'll indeed require a major refactor 🙂. 

I did create an example where I use a combination of `onMouseMove` /`onMouseUp`/`onMouseDown` and `onWheel` to get the mouse position and update the item's `top` CSS property accordingly. The example can be seen [here](https://codesandbox.io/s/timeline-demo-move-vertical-position-matches-users-mouse-position-bdorh). 

I'm aware it's kinda hacky, but I hope this is helpful for those that absolutely need this behavior.

Thanks for your work in this lib 👍.
